### DISCUSSION TITLE
Button inside frame removes all content

### DISCUSSION THREAD

**Discussion by Grmglib** [dotnet/maui#17108] [Q&A]
When i add a button to a grid inside a frame it becomes gray.
![image](https://github.com/dotnet/maui/assets/82120970/62c37989-832f-4d17-8948-25f8d874cc93)
![image](https://github.com/dotnet/maui/assets/82120970/8f016c75-880b-47f7-9aec-10f5ec9d473b)
When i remove the button
![image](https://github.com/dotnet/maui/assets/82120970/7f9bd5c7-1e57-4cac-ab24-690d0ebe00bd)
![image](https://github.com/dotnet/maui/assets/82120970/9559eab9-a02f-4621-a107-49492bfd6b32)

**Comment by bradencohen** [comment]
Can you use [Border](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/border)?
